### ISSUE TITLE
MonitorEventsAsync reports too little information

### ISSUE THREAD

**Issue by thoemmi** [Microsoft/Docker.DotNet#310]
**Output of `dotnet --info`:**

```
Docker version 18.05.0-ce-rc1, build 33f00ce
```

**What version of Docker.DotNet?:**

```
3.125.2
```

Using `MonitorEventsAsync`, the `JSONMessage` passed to `IProgress.Report` provides to little details.

E.g. this is the original JSON (reformatted to increase readbility):
```javascript
{
  "status": "tag",
  "id": "sha256:6c367cf4cb9815b10e47545dc9539ee4bd5cd0f8697d33f4d9cb1e1850546403",
  "Type": "image",
  "Action" :"tag",
  "Actor": {
    "ID": "sha256:6c367cf4cb9815b10e47545dc9539ee4bd5cd0f8697d33f4d9cb1e1850546403",
    "Attributes": {   
      "name": "microsoft/nanoserver:MonitorTests"
    }
  },
  "scope": "local",
  "time": 1524774401,
  "timeNano": 1524774401646293400
}
```
However, the `JSONMessage` instance is filled with these properties:
```
Aux = null
Error = null
ErrorMessage = null
From = null
ID = "sha256:6c367cf4cb9815b10e47545dc9539ee4bd5cd0f8697d33f4d9cb1e1850546403"
Progress = null
ProgressMessage = null
Status = "tag"
Stream = null
Time = {26.04.2018 20:26:41}
TimeNano = 1524774401646293400
```
Other important information like the type or the tag name is missing.

E.g. if you receive a `JSONMessage` with `Status = "create"`, you don't know whether a container, an image or something else was created.

**Comment by thoemmi** [comment]
Hello @jterry75, I tried it fix it myself, but didn't succeed: I installed go (go1.10.1 windows/amd64), but when calling `update-generated-code.cmd` it cannot find the packages `github.com/docker/docker/api/types` etc. I guess it has something to do with `GOPATH`, `src`, and `vendor`. As you might already guess I'm not familiar with go 😉 

TL;DR: I guess it's more conducive if you fix it.
